### PR TITLE
Add Scan Edge later for spark

### PR DESCRIPTION
**Pull request by critical27**
1. Add basic data encapsulation
2. Add scan edge method
3. Add basic test skeleton

@darionyaphet, you can refer to `StorageScanEdgeExample` (it maybe can move to Examples module later).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +0,0 @@
-language: java
```
Comment (darionyaphet) [inline .travis.yml]: why remove this ？It's useful.

File: client/src/test/java/com/vesoft/nebula/data/NebulaCodecTest.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
```
Comment (darionyaphet) [inline client/src/test/java/com/vesoft/nebula/data/NebulaCodecTest.java]: 2019

File: .travis.yml (hunk 1 above)
Comment (critical27) [reply .travis.yml]: 👌

File: client/pom.xml [hunk 3]
```diff
@@ -182,5 +200,10 @@
             <artifactId>commons-codec</artifactId>
             <version>1.13</version>
         </dependency>
+        <dependency>
+            <groupId>com.vesoft</groupId>
+            <artifactId>nebula-utils</artifactId>
+            <version>1.0.0-beta</version>
```
Comment (darionyaphet) [inline client/pom.xml]: RC2 ?

File: client/src/main/java/com/vesoft/nebula/meta/client/MetaClientImpl.java [hunk 4]
```diff
@@ -6,6 +6,8 @@
 
 package com.vesoft.nebula.meta.client;
 
+import static com.vesoft.nebula.utils.NebulaCommon.LATEST_SCHEMA_VERSION;
```
Comment (darionyaphet) [inline client/src/main/java/com/vesoft/nebula/meta/client/MetaClientImpl.java]: nice

File: client/src/main/java/com/vesoft/nebula/storage/client/StorageScanEdgeExample.java [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class StorageScanEdgeExample {
+    private static final Logger LOGGER = LoggerFactory.getLogger(StorageScanEdgeExample.class);
+    private static MetaClientImpl metaClient;
```
Comment (darionyaphet) [inline client/src/main/java/com/vesoft/nebula/storage/client/StorageScanEdgeExample.java]: MetaClient and StorageClien

Comment (critical27) [reply client/src/main/java/com/vesoft/nebula/storage/client/StorageScanEdgeExample.java]: Ok

File: client/pom.xml (hunk 3 above)
Comment (critical27) [reply client/pom.xml]: Good catch, wait for `nebula-utils` update

**Pull request merged**